FvwmPager: Division by zero in x_pager.c

Thanks for reporting your bug here!  The following template will help with
giving as much information as possible so that it's easier to diagnose and
fix.

## Upfront Information

Please provide the following information by running the command and providing
the output.

* Fvwm3 version (run: `fvwm3 --version`)
fvwm3 1.0.3 (1.0.2-44-g812b3964-dirty)
* Linux distribution or BSD name/version
openSUSE 15.2 in Windows Subsystem for Linux
* Platform (run: `uname -sp`)
Linux x86_64
## Expected Behaviour
FvwmPager shows up in the right panel.
What were you trying to do?  Please explain the problem.
Ran Cppcheck 2.3 on fvwm3. It shows there is a division by zero in `modules/FvwmPager/x_pager.c` if `Rows` < 0 and `Columns` == 0. Then I copied the default fvwm3 config to ~/.fvwm/config and read the .adoc for FvwmPager to see how to specify Rows and Columns values.



## Actual Behaviour

What should have happened, but didn't?
FvwmPager should've showed up in the right panel but it doesn't with the two-line tweak to the default fvwm3 config

## Enabling logging

`fvwm3` has a means of logging what it's doing.  Enabling this when
reproducing the issue might help.  To do this, either change the means fvwm3
is started by adding `-v` as in:

```
fvwm3 -v


```

or, once `fvwm3` has loaded, send `SIGUSR2` as in:

```
pkill -USR2 fvwm3
```

The resulting logfile can be found in `$HOME/.fvwm/fvwm3-output.log`

## Steps to Reproduce

How can the problem be reproduced?  For this, the following is helpful:
1. Install fvwm3
2. Start fvwm3
3. Left click on the background somewhere and click Copy Config
4. Click confirm
5. Open ~/.fvwm/config
6. Add these two lines after the `*FvwmPager: ...` lines after `DestroyModuleConfig FvwmPager:*`
```
*FvwmPager: Rows -1
*FvwmPager: Columns 0
```
7. Restart fvwm3
* Reduce the problem to the smallest `fvwm` configuration example (where
  possible).  Start with a blank config file (`fvwm3 -f/dev/null`) and go from
  there.

* Does the problem also happen with Fvwm2?
Yes.
Include your configuration with this issue.
[config.txt](https://github.com/fvwmorg/fvwm3/files/5891077/config.txt)
[minimal-config.txt](https://github.com/fvwmorg/fvwm3/files/5891241/minimal-config.txt) (only StartFunction, RightPanel, and FvwmPager config sections included)

## Does Fvwm3 crash?
No. The FvwmPager module doesn't show up though if I add the above two lines to the default fvwm3 config.

If fvwm3 crashes then check your system for a corefile.  This is platform
dependant, however, check if:

  - corefiles are enabled (`ulimit -c`)
  - A corefile might be in `$HOME` or `/tmp/`.
  - If you're using Linux (with `systemd`), check `coredumpctl list`.
    `coredumpctl` may need installing separately.

If you find a corefile, install `gdb` and run:

```
gdb /path/to/fvwm3 /path/to/corefile
```

If you're using `coredumpctl` then use:

```
coredumpctl debug
```

Then from within the `(gdb) ` prompt, issue:

```
bt full
```

... and include the output here.

## Extra Information

* Anything else we should know?
This can be fixed by changing [line 552](https://github.com/fvwmorg/fvwm3/blob/812b39646e5c5dab303cded43d6eb9a11e8f0a81/modules/FvwmPager/x_pager.c#L552) of x_pager.c to `if (Columns <= 0)` instead of `if (Columns < 0)`
* Feel free to take a screen capture or video and upload to this issue if you
  feel it would help.
![VcXsrv Server - Display MSI_0 0 2021-01-28 5_57_28 PM](https://user-images.githubusercontent.com/38995150/106221394-6571ac00-6192-11eb-8db8-0f12688fb83b.png)


* Attach `$HOME/.fvwm/fvwm3-output.log` from the step above.

[fvwm3-output.log](https://github.com/fvwmorg/fvwm3/files/5891051/fvwm3-output.log)

Cppcheck 2.3 results for x_pager.c:

`> /mnt/d/Cppcheck/cppcheck.exe --enable=warning,performance,portability modules/FvwmPager/x_pager.c`
```
Checking modules\FvwmPager\x_pager.c ...
modules\FvwmPager\x_pager.c:956:27: warning: Obsolete function 'alloca' called. In C99 and later it is recommended to use a variable length array instead. [allocaCalled]
    shape = (XRectangle *)alloca (shape_count * sizeof (XRectangle));
                          ^
modules\FvwmPager\x_pager.c:559:20: warning: Either the condition 'Columns<0' is redundant or there is division by zero at line 559. [zerodivcond]
      Rows = ndesks/Columns;
                   ^
modules\FvwmPager\x_pager.c:552:16: note: Assuming that condition 'Columns<0' is not redundant
    if(Columns < 0)
               ^
modules\FvwmPager\x_pager.c:559:20: note: Division by zero
      Rows = ndesks/Columns;
                   ^
Checking modules\FvwmPager\x_pager.c: DEBUG...
```

--- a/modules/FvwmPager/x_pager.c
+++ b/modules/FvwmPager/x_pager.c
@@ -549,7 +549,7 @@ void initialize_pager(void)
   /* Size the window */
   if(Rows < 0)
   {
-    if(Columns < 0)
+    if(Columns <= 0)
     {
       Columns = ndesks;
       Rows = 1;

